cluster-service/deploy/helm/templates/cluster-proxy-service-config.configmap.yaml

```diff
@@ -6,12 +6,3 @@ metadata:
   namespace: {{ .Release.Namespace }}
 data:
   config.yaml: |
-    # Hosts that should be added to noProxy for all clusters
-    noProxy: []
-    # Hosts that should be added to noProxy for AWS clusters
-    noProxy_aws: []
-    # Hosts that should be added to noProxy for GCP clusters
-    noProxy_gcp: []
-    # Readiness endpoints that verify proxy connectivity
-    readinessEndpoints:
-      - "https://api.openshift.com"
```

Comment:
cc @miguelsorianod

Reply:
This should be reverted.  It's not related to the ARM API.